### DISCUSSION TITLE
Error searching messages, API responded with status 404! 	{"message":"Unknown Guild","code":10004} CoreException	{}

### DISCUSSION THREAD

**Discussion by unsainted-calvery** [victornpb/undiscord#654] [1. Questions & Answers]
![Screenshot 2024-08-12 220706](https://github.com/user-attachments/assets/54764f70-1b28-4755-90a4-e49cefc52b78)

I am filling everything out correctly and its giving me this error. For context I am trying to delete a bots message in a dm so I copied the bots user id and put it in the authors ID, then copied and pasted the numbers after the @me on the browser URL for the server ID, then copied the dms channel ID and pasted it in the channel ID. I also filled out the filter searching with the text i wanted to filter and checked off the has: link box and it gave that error. I have no idea why, if anyone has any answers I would appreciate it thanks.

**Comment by TiffanyBlitz** [comment]
I am getting the same response.

I am only trying to delete my own messages from a DM.

As instructed, I have "@me" as the SERVER ID, and the CHANNEL ID is blank.

Forgive me if I'm missing something obvious - new around here.

**Reply by LittleMissCrazyGirl** [reply]
Hello! I was trying to use the script for the first time today and received the same response. I was able to fix it by also clicking the fill button on the Authorization Token button. However, after I did that I received a new error number 400 which I resolved by removing my email address from the Messages interval filter (I am not sure how it got there. I think it happens by default.). It is now working as it should. I hope this information helps you.

**Comment by TiffanyBlitz** [comment]
I appreciate the response!

Between those, and in desperation clicking to let it fill the Channel ID
(even though the instructions said to leave it blank for DMs), I got it
working.

Thanks so much for your help!

**Reply by LittleMissCrazyGirl** [reply]
Sure! I also noticed if you set the delay times too fast it will auto quit after a bit.

**Comment by PREMON1TION** [comment]
What helped me, is to increase the Search & Delete Delay. With these settings it works for me without error:
Search Delay: 30000ms
Delete Delay: 800ms
